DemoApp/DemoApp/DemoApp.csproj

```diff
@@ -84,6 +84,9 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <Compile Update="Views\TableViewPage.xaml.cs">
```

Comment:
Performance is very poor when we have too many records - like 100.

Reply:
Created a new ticket for this issue, #192